tsdb/agent/db_test.go

```diff
@@ -333,8 +333,8 @@ func TestRollback(t *testing.T) {
 		}
 	}
 
-	// Check that the rollback ensured nothing got stored.
-	require.Equal(t, 0, walSeriesCount, "series should not have been written to WAL")
+	// Check that only series get stored after calling Rollback.
+	require.Equal(t, numSeries*3, walSeriesCount, "series should not have been written to WAL")
```

Comment:
I'm not exactly sure where `*3` comes from, but it's also used in the tests above.